src/web/lib/ArticleRenderer.tsx

```diff
@@ -235,7 +236,18 @@ export const ArticleRenderer: React.FC<{
                         </div>
                     );
                 case 'model.dotcomrendering.pageElements.RichLinkBlockElement':
-                    return <div key={i} id={`rich-link-${i}`} />;
+                    return (
+                        <div key={i} id={`rich-link-${i}`}>
+                            <DefaultRichLink
+                                index={i}
+                                headlineText={element.text}
+                                url={element.url}
+                                isPlaceholder={true}
+                            />
+                        </div>
+                    );
+
+                // return <div key={i} id={`rich-link-${i}`} />;
```

Comment:
Don't forget to remove!

Reply:
doh, will raise a quick PR to do this. Thanks.